platform/public/css/index.css

```diff
@@ -21,4 +21,20 @@
 .button-dark {
     background-color: darkslategrey;
     color: aliceblue;
-}
\ No newline at end of file
+}
+
+#feedback-button {
+    position: absolute !important;
+    z-index: 10000 !important;
+    padding: 12px;
+    border-radius: 20px;
+    box-shadow: 0 4px 8px 0 gray, 0 6px 20px 0 gray;
+    background-color: lightblue;
+    bottom: 5%;
+    right: 2%;
```

Comment:
This places this on top of content in the learning activity, which isn't ideal. Could we place it in the bottom-**left** corner instead?